app/localoperator/DieterichRuinaAgeing.h

```diff
@@ -99,19 +99,20 @@ class DieterichRuinaAgeing {
                 if (a > b) {
                     std::swap(a, b);
                 }
-                auto fF = [this, &index, &snAbs, &tauAbs, &psi, &eta](double V) {
-                    return tauAbs - this->F(index, snAbs, V, psi) - eta * V;
+                auto fF = [this, &index, &snAbs, &tauAbs, &psi, &eta](double Ve) {
+                    return tauAbs - this->F(index, snAbs, std::pow(10.0,Ve), psi) - eta * std::pow(10.0,Ve);
                 };
                 try {
-                    V = zeroIn(a, b, fF);
+                    auto Ve = zeroIn(-216, std::log10(b), fF);
```

Comment:
What about this hardcoded -216?
Do you mean the smallest double? (1e−308 or so?).
Maybe increasing this number a bit could save you a few iterations? (e.g. -50 or so?).

Reply:
I removed the hardcoded -216, which was somewhat of a random choice after -32 did not work. Now, I implemented a loop to apply a = -2^-5 ... -2^-8, which saves some more time (I'm currently running the benchmark test).